packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -430,6 +447,18 @@ class AndroidCameraCameraX extends CameraPlatform {
     captureOrientationLocked = false;
   }
 
+  /// Sets the exposure point for automatically determining the exposure values.
+  ///
+  /// Supplying `null` for the [point] argument will result in resetting to the
+  /// original exposure point value.
+  ///
+  /// [cameraId] is not used.
```

Comment:
Because it's part of the platform interface 😅 I think, though, it is included in all/most methods in the plugin for the purpose of creating various camera event streams. I plan to clarify this when fixing https://github.com/flutter/flutter/issues/126270.